### PR TITLE
Local installation docu update

### PR DESCRIPTION
**Pull request by tomekpapiernik**
Confirm these statements before you submit your pull request:

- [x] I have read and submitted the required [Contributor Licence Agreements](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#agreements-and-licenses).
- [x] This pull request follows the contributing guidelines, recommended Git workflow, and templates.
- [ ] I have tested my changes.
- [x] I have updated the relevant documentation.
---

**Description**

Changes proposed in this pull request:

- Updated local installation prerequisites
- Removing the "prerequisites reasoning" document and moving the relevant content to the main document
- Moving the installer CR description to a separate document

### CODE REVIEW

File: docs/kyma/docs/031-gs-local-installation.md [hunk 1]
```diff
@@ -15,35 +15,38 @@ To run Kyma locally, clone this Git repository to your local machine and downloa
...
 - [Helm](https://github.com/kubernetes/helm) 2.8.2
 - [jq](https://stedolan.github.io/jq/)
-- [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac only
-- [Virtualbox](https://www.virtualbox.org/) - Linux or Windows
+
+Vitrualization:
+
+- [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac
+- [Virtualbox](https://www.virtualbox.org/) - Linux, Windows, Mac
 - Hyper-V - Windows
```
Comment (derberg) [inline docs/kyma/docs/031-gs-local-installation.md]: maybe add link to https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v ?

File: docs/kyma/docs/031-gs-local-installation.md [hunk 2]
```diff
@@ -61,30 +64,9 @@ scripts/is-installed.sh
 Read the [Reinstall Kyma](025-details-local-reinstallation.md) document to learn how to reinstall Kyma without deleting the cluster from Minikube.
 To learn how to test Kyma, see the [Testing Kyma](026-details-testing.md) document.
 
-### Custom Resource file
+### Customize the Kyma installation
```
Comment (derberg) [inline docs/kyma/docs/031-gs-local-installation.md]: I think that this is now a note and not a section

File: docs/kyma/docs/031-gs-local-installation.md [hunk 3]
```diff
@@ -61,30 +64,9 @@ scripts/is-installed.sh
...
-- `--url` - the URL of the Kyma package to install
-- `--version` - the Kyma version
-- `--ipaddr` - the load balancer IP
-- `--domain` - the instance domain
-
-For example:
-```
-$ ./installation/scripts/create-cr.sh --output kyma-cr.yaml --url {Kyma_TAR.GZ_URL} --version 0.0.1 --domain kyma.local
-```
+Read the [Installer Custom Resource](./037-CRD.md) document to learn how to customize the Kyma installation by customizing the installer Custom Resource file.
```
Comment (derberg) [inline docs/kyma/docs/031-gs-local-installation.md]: remember that for now links still do not work in the docs UI so please remove it

File: docs/kyma/docs/037-CRD.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+title: Installer Custom Resource
+type: CRD
+---
+
+The installer Custom Resource (CR) file controls the Kyma installer, which is a proprietary solution based on the [Kubernetes operator](https://coreos.com/operators/). The file contains the basic information that defines Kyma installation. You can generate a custom CR file to install Kyma using specific parameters or a specific release in a `TAR.GZ` format.
+Find the custom resource template [here](../../../installation/resources/installer-cr.yaml.tpl).
```
Comment (derberg) [inline docs/kyma/docs/037-CRD.md]: remember that for now links still do not work in the docs UI, especially link that point outside the docu folder, so please include the code snippet instead

File: docs/kyma/docs/037-CRD.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+title: Installer Custom Resource
+type: CRD
```
Comment (derberg) [inline docs/kyma/docs/037-CRD.md]: if this is CRD, which btw I think it is a very good idea
this should follow same structure as the other CRD you already created

File: docs/kyma/docs/037-CRD.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+title: Installer Custom Resource
```
Comment (derberg) [inline docs/kyma/docs/037-CRD.md]: the name of crd is Installation, so please us it in the title

File: docs/kyma/docs/037-CRD.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+The installer Custom Resource (CR) file controls the Kyma installer, which is a proprietary solution based on the [Kubernetes operator](https://coreos.com/operators/). The file contains the basic information that defines Kyma installation. You can generate a custom CR file to install Kyma using specific parameters or a specific release in a `TAR.GZ` format.
+Find the custom resource template [here](../../../installation/resources/installer-cr.yaml.tpl).
+
+## Control the installation process
+
+To trigger the installation process, set the **action** label to `install` in the metadata of the Custom Resource with the installer configuration.
+To trigger the deinstallation process, set the **action** label to `uninstall` in the metadata of the Custom Resource with the installer configuration.
+
+## Generate a new Custom Resource file
```
Comment (derberg) [inline docs/kyma/docs/037-CRD.md]: there is no explanation of finalizers
https://github.com/kyma-project/kyma/blob/a93fe9f83f420f2bfeb0957f34d41f087df9b3d2/installation/resources/installer-cr.yaml.tpl#L7

File: docs/kyma/docs/037-CRD.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+- `--output` - mandatory. The location of the Custom Resource output file
+- `--url` - the URL of the Kyma package to install
+- `--version` - the Kyma version
+- `--ipaddr` - the load balancer IP
+- `--domain` - the instance domain
+
+For example:
+```
+$ ./installation/scripts/create-cr.sh --output kyma-cr.yaml --url {Kyma_TAR.GZ_URL} --version 0.0.1 --domain kyma.local
```
Comment (derberg) [inline docs/kyma/docs/037-CRD.md]: you need to be more specific in URL part, the only official link we can provide is the one located here: https://github.com/kyma-project/kyma/releases/tag/latest

which is https://github.com/kyma-project/kyma/archive/latest.tar.gz
I don't see other option

File: docs/kyma/docs/031-gs-local-installation.md [hunk 9]
```diff
@@ -15,35 +15,38 @@ To run Kyma locally, clone this Git repository to your local machine and downloa
...
+Vitrualization:
+
+- [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac
+- [Virtualbox](https://www.virtualbox.org/) - Linux, Windows, Mac
 - Hyper-V - Windows
 
-Read the [prerequisite reasoning](019-prereq-reasoning.md) document to learn why Kyma uses these tools.
+> **NOTE:** To work with Kyma, use only the provided installation and deinstallation scripts. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command. If you do not need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
 
 ## Setup certificates
```
Comment (derberg) [inline docs/kyma/docs/031-gs-local-installation.md]: doesn't seem like you took into account comment from Andreas about installing certificate on linux

File: docs/kyma/docs/031-gs-local-installation.md (hunk 1 above)
Comment (tomekpapiernik) [reply docs/kyma/docs/031-gs-local-installation.md]: sure

File: docs/kyma/docs/040-installation-custom-resource.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+spec:
+  version: "1.0.0"
+  url: "https://sample.url.com/kyma_release.tar.gz"
+```
+
+This table analyses the elements of the sample CR and the information it contains:
+
+| Field   |      Mandatory?      |  Description |
+|:----------:|:-------------:|:------|
+| **apiVersion** | **YES** | Defined basing on the `group` and `version` fields of the CRD `spec` section. |
```
Comment (klaudiagrz) [inline docs/kyma/docs/040-installation-custom-resource.md]: Shouldn't `group`, `version`, and `spec` be in bold?

File: docs/kyma/docs/040-installation-custom-resource.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+| **kind** | **YES** | Defined basing on the `names: kind` field of the CRD `spec` section. |
+| **metadata.name** | **YES** | Specifies the name of the CR. |
+| **metadata.labels.action** | **YES** | Defines the behavior of the Kyma installer. Available options: `install` and `uninstall`. |
```
Comment (klaudiagrz) [inline docs/kyma/docs/040-installation-custom-resource.md]: ``Available options are `install` and `uninstall`.`` -> full sentences are always better 😉

File: docs/kyma/docs/031-gs-local-installation.md [hunk 12]
```diff
@@ -15,33 +15,39 @@ To run Kyma locally, clone this Git repository to your local machine and downloa
...
 - [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac only
 - [VirtualBox](https://www.virtualbox.org/) - Linux or Windows
-- Hyper-V - Windows
+- [Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v) - Windows
+
+> **NOTE:** To work with Kyma, use only the provided installation and deinstallation scripts. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command. If you don't need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
 
 ## Set up certificates
 
 Kyma comes with a local wildcard self-signed `server.crt` certificate that you can find under the `/installation/certs/workspace/raw/` directory of the `kyma` repository. Trust it on the OS level for convenience. Alternatively, accept exceptions for each subdomain in your browser as you use Kyma.
```
Comment (pbochynski) [inline docs/kyma/docs/031-gs-local-installation.md]: Please remove "Alternatively, accept exceptions for each subdomain in your browser as you use Kyma." or explain that you need to hit every host added to etc/hosts (more than 10)

**Pull request merged**